docs/technical_reference/property_models/NaCl_T_dep.rst

```diff
@@ -0,0 +1,120 @@
+   "Vapor Pressure", ":math:`P_sat`", "pressure_sat", "None", ":math:`\text{MPa}`"
+   "Dynamic viscosity", ":math:`\mu`", "visc_d_phase", "[p]", ":math:`\text{Pa/s}`"
+   "Thermal Conductivity", ":math:`k`", "therm_cond_phase", "[p]", ":math:`\text{W/m*K}`"
+   "Specific Heat Capacity", ":math:`C_p`", "cp_mass_phase", "[p]", ":math:`\text{J/kg*K}`"
+   "Solute diffusivity", ":math:`D`", "diffus_phase_comp", "[p,'NaCl']", ":math:`\text{m^2/s}`"
+   "Osmotic coefficient", ":math:`\phi`", "osm_coeff", "None", ":math:`\text{dimensionless}`"
+   "Osmotic pressure", ":math:`\pi`", "pressure_osm", "None", ":math:`\text{Pa}`"
+   "Specific enthalpy", ":math:`\widehat{H}`", "enth_mass_phase", "[p]", ":math:`\text{kJ/kg}`"
+   "Enthalpy flow", ":math:`H`", "enth_flow", "None", ":math:`\text{J/s}`"
+
+Relationships
+-------------
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Component mass fraction", ":math:`X_j = \frac{M_j}{\sum_{j} M_j}`"
+   "Mass density", "Equation 7 in Sparrow (2003)"
+   "Volumetric flowrate", ":math:`Q = \frac{\sum_{j} M_j}{\rho}`"
+   "Mass concentration", ":math:`C_j = X_j \rho`"
+   "Component mole flowrate", ":math:`N_j = \frac{M_j}{MW_j}`"
+   "Component mole fraction", ":math:`y_j = \frac{N_j}{\sum_{j} N_j}`"
+   "Molality", ":math:`Cm_{NaCl} = \frac{x_{NaCl}}{(1-x_{NaCl}) MW_{NaCl}}`"
+   "Solubility", "Equation 5 in Sparrow (2003)"
+   "Vapor Pressure", "Equation 6 in Sparrow (2003)"
+   "Dynamic viscosity", "Regressed from Zaytsev & Aseev (1992)"
+   "Thermal Conductivity", "Regressed from Zaytsev & Aseev (1992)"
+   "Specific Heat Capacity", "Regressed from Zaytsev & Aseev (1992)"
+   "Solute diffusivity", "Regressed from Zaytsev & Aseev (1992)"
+   "Osmotic coefficient", "Regressed from Pitzer et. al. (1984)"
+   "Osmotic pressure", ":math:`\pi = i \phi Cm_{NaCl} \rho_w R T` [See note 1 below]"
+   "Specific enthalpy", "Equation 8 in Sparrow (2003)"
+   "Enthalpy flow", ":math:`H = \sum_{j} M_j \widehat{H}`"
+
+Note 1: Osmotic pressure calculation uses the van 't Hoff factor (:math:`i\text{, assumed to be 2}`), density of water (as a function of temperature), gas constant (:math:`R\text{, 8.314 J/mol}\text{K}`) in addition to previously defined variables.
+
+
+Scaling
+-------
+This NaCl property package includes support for scaling, such as providing default or calculating scaling factors for almost all variables. The only variables that do not have scaling factors are the component mass flowrate and the user must set them or the user will receive a warning.
+
+The user can specify the scaling factors for component mass flowrates with the following:
+
+The default scaling factors are as follows:
+
+   * 1e-2 for temperature
+   * 1e-6 for pressure
+   * 1e-3 for mass density
+   * 1e3 for dynamic viscosity
+   * 1e9 for solute diffusivity
+   * 1 for the osmotic coefficient
+   * 1e-2 for the specific enthalpy
+   * 1e-4 for the specific heat capacity
+   * 1e2 for vapor pressure
```

Comment:
revisit this considering the changes in relationships and requested changes from MPa to Pa